src/streaming/AudioConfig.py

```diff
@@ -0,0 +1,29 @@
+"""
+    Configuration for an audio stream
+"""
+
+class AudioConfig():
+    def __init__(self, content_type = 'audio/*', layout = None, rate = None, audio_format = None, channels = None):
+        """Constructor for AudioConfig class
+
+        :param content_type (optional): string containing content type. Default will resort to 
+        :param layout (optional): layout of audio
+        :param rate (optional): sampling rate of audio
+        :param audio_format (optional): format of audio
+        :param channels (optional): the number of channels the audio has
+        """
+        self. content_type = content_type
+        self.layout = layout
+        self.rate = rate
+        self.format = audio_format
+        self.channels = channels
+
+    def get_content_type_string(self):
+        """Returns the content type and params as a string for the websocket
+           connection
+        """
+        return self.content_type +  \
```

Comment:
Extra space after the `+`

Reply:
@kbridbur  There are python linters. Do we use them?

Reply:
`flake8 --ignore=E122,E126,E127,E501`

Reply:
https://speech-knowledge.rev.com/intro/coding-conventions

Reply:
There are and I believe we use flake8. I'll double check